platform/android/export/export.cpp

```diff
@@ -757,6 +757,32 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return OK;
 	}
 
+	static Error copy_gradle_so(void *p_userdata, const SharedObject &p_so) {
```

Comment:
Seems like you can just pass a `bool` parameter instead of the `void *p_userdata` parameter.

Reply:
The function signature is defined in `editor/editor_export.h`:
https://github.com/godotengine/godot/blob/c8444c3ee078e33c33287cf879aa5daecb962a80/editor/editor_export.h#L169

Reply:
`CustomExportData` is now passed as user data to store copied file data.